### PR TITLE
fix(doris) , mysql group_concat function different doris , it is not support separator keyword

### PR DESCRIPTION
**Pull request by Toms1999**

### CODE REVIEW

File: sqlglot/dialects/doris.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ class Generator(MySQL.Generator):
             exp.ArrayUniqueAgg: rename_func("COLLECT_SET"),
             exp.CurrentTimestamp: lambda self, _: self.func("NOW"),
             exp.DateTrunc: lambda self, e: self.func("DATE_TRUNC", e.this, unit_to_str(e)),
+            exp.GroupConcat: lambda self,e: exp.func("GROUP_CONCAT", e.this, e.args.get("'seprator'") or exp.Literal.string(",")),
```
Comment (tobymao) [inline sqlglot/dialects/doris.py:52]: this is wrong, and not tested, " 'separator' " has extra quotes. 

can you please fix this and double check your work next time? if these low effort prs continue, we will no longer accept future contributions from you.

File: tests/dialects/test_doris.py [hunk 2]
```diff
@@ -48,6 +48,13 @@ def test_doris(self):
                 "postgres": """SELECT JSON_EXTRACT_PATH(CAST('{"key": 1}' AS JSONB), 'key')""",
             },
         )
+        self.validate_all(
+            """SELECT GROUP_CONCAT('aa', ',')""",
+            read={
+                "postgres": """SELECT STRING_AGG('aa', ',')""",
+                "mysql": """SELECT GROUP_CONCAT('aa' SEPARATOR ',')""",
+            }
+        )
```
Comment (georgesittas) [inline tests/dialects/test_doris.py:57]:
```suggestion
        self.validate_all(
            "SELECT GROUP_CONCAT('aa', ',')",
            read={
                "doris": "SELECT GROUP_CONCAT('aa', ',')",
                "mysql": "SELECT GROUP_CONCAT('aa' SEPARATOR ',')",
                "postgres": "SELECT STRING_AGG('aa', ',')",
            }
        )
```

File: sqlglot/dialects/doris.py [hunk 3]
```diff
@@ -49,6 +49,7 @@ class Generator(MySQL.Generator):
             exp.ArrayUniqueAgg: rename_func("COLLECT_SET"),
             exp.CurrentTimestamp: lambda self, _: self.func("NOW"),
             exp.DateTrunc: lambda self, e: self.func("DATE_TRUNC", e.this, unit_to_str(e)),
+            exp.GroupConcat: lambda self,e: exp.func("GROUP_CONCAT", e.this, e.args.get("seprator") or exp.Literal.string(",")),
```
Comment (georgesittas) [inline sqlglot/dialects/doris.py:52]:
```suggestion
            exp.GroupConcat: lambda self, e: exp.func(
                "GROUP_CONCAT", e.this, e.args.get("seprator") or exp.Literal.string(",")
            ),
```

**Comment by georgesittas** [comment]
@Toms1999 please run `make style` before submitting future PRs.

File: sqlglot/dialects/doris.py [hunk 4]
```diff
@@ -49,6 +49,9 @@ class Generator(MySQL.Generator):
             exp.ArrayUniqueAgg: rename_func("COLLECT_SET"),
             exp.CurrentTimestamp: lambda self, _: self.func("NOW"),
             exp.DateTrunc: lambda self, e: self.func("DATE_TRUNC", e.this, unit_to_str(e)),
+            exp.GroupConcat: lambda self, e: exp.func(
```
Comment (georgesittas) [inline sqlglot/dialects/doris.py:52]:
```suggestion
            exp.GroupConcat: lambda self, e: self.func(
```

File: sqlglot/dialects/doris.py [hunk 5]
```diff
@@ -49,6 +49,9 @@ class Generator(MySQL.Generator):
...
+                "GROUP_CONCAT", e.this, e.args.get("seprator") or exp.Literal.string(",")
```
Comment (georgesittas) [inline sqlglot/dialects/doris.py:53]:
```suggestion
                "GROUP_CONCAT", e.this, e.args.get("separator") or exp.Literal.string(",")
```

**Pull request merged**